packages/web-components/fast-element/package.json

```diff
@@ -20,8 +22,17 @@
   "exports": {
     ".": {
       "types": "./dist/fast-element.d.ts",
-      "default": "./dist/esm/index.js"
-
+      "default": "./dist/esm/index.js",
```

Comment:
I don't know if this is actually impacting anything, but per https://nodejs.org/api/packages.html#conditional-exports the `default` field should come last.